### PR TITLE
📖 Minor typos and copy-editing to checks/write.md

### PR DESCRIPTION
**Pull request by pnacht**
#### What kind of change does this PR introduce?

Fixes a few typos and improves legibility in write.md.

### CODE REVIEW

File: checks/write.md [hunk 1]
```diff
@@ -19,65 +19,63 @@ criteria and then create a new GitHub Issue to discuss with the team:
     of a given scorecard finding for that repository.
 -   Any criteria in the scorecard must be actionable. It should be possible,
     with help, for any project to "check all the boxes".
--   Any solution to compile a scorecard should be usable by the greater open
-    source community to monitor upstream security.
+-   Any solution to compile a scorecard should be usable by the greater
+    open-source community to monitor upstream security.
```
Comment (olivekl) [inline checks/write.md:23]: Open style question here, whether to hyphenate open source as a compound adjective before a noun. Popular usage is trending toward dropping the hyphen, and there's little chance of confusion (nobody will misread it as open source-community).

File: checks/write.md [hunk 2]
```diff
@@ -19,65 +19,63 @@ criteria and then create a new GitHub Issue to discuss with the team:
...
+    *   For maximum and minimum scores, use `checker.CreateMaxScoreResult()` and
+        `checker.CreateMinScoreResult()`, respectively.
     *   If you need more flexibility and need to set a specific score, use
         `checker.CreateResultWithScore()` with one of the constants declared,
         such as `checker.HalfResultScore`.
 
 6.  Dealing with errors: see [errors/errors.md](/errors/errors.md).
 
-7.  Create unit tests for both low, high and inconclusive score. Put them in a
+7.  Create unit tests for both low, high and inconclusive scores. Put them in a
```
Comment (olivekl) [inline checks/write.md:72]: Can we also remove "both" here (since there are three)?

**Review by olivekl — COMMENTED** [review]
Nice changes, thank you!

File: checks/write.md (hunk 1 above)
Comment (pnacht) [reply checks/write.md:23]: Sure thing. I actually thought this was a mandatory hyphen, not a stylistic choice. Learn something new every day.

**Review by olivekl — APPROVED** [review]

**Pull request merged**